### PR TITLE
S2 game screen

### PR DESCRIPTION
**Pull request by cr-walton**
Story 2 for trainer review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  <title>Quotlefish - A movie game</title>
+</head>
+<body>
+	<header>
+		<div class="headerContainer">
+			<div>
+				<h1>Quotlefish</h1>
+			</div>
+			<img src="assets/info.png" id="instructionsModalBtn"/>
+			<!-- The Modal -->
```
Comment (charliecog) [inline index.html]: remove comment - id and class name explains it just fine

File: index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+			<div id="instructionsModal" class="instructionsModal">
+				<!-- Modal content -->
```
Comment (charliecog) [inline index.html]: remove comment - id and class name explains it just fine

File: index.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+				<div class="modal-content">
```
Comment (charliecog) [inline index.html]: inconsistent naming format - hyphens used here, (check whole document for this)

File: quiz.html [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```
Comment (charliecog) [inline quiz.html]: delete this file, doesnt appear to be used

File: style.css [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+
+body {
+    margin: 0;
+    font-family: Arial, 'Open Sans';
+    font-size: 16px;
+    text-align: center;
+    background-image: url('assets/background_image_Quizshow.jpg');
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 100%;
```
Comment (charliecog) [inline style.css]: possibly could be removed

File: style.css [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+
+main {
+    width: 357px;
```
Comment (charliecog) [inline style.css:20]: super specific, perhaps a percentage like 40% instead?

File: style.css [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+    width: 100%;
+    align-items: center;
+}
+
+.headerContainer {
+    width: 357px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
```
Comment (charliecog) [inline style.css]: remove line break

File: style.css [hunk 8]
```diff
@@ -0,0 +1,171 @@
...
+    display: flex;
+    justify-content: center;
+    background-color: #ffffff;
+    height: 8vh;
+    width: 100%;
+    align-items: center;
+}
+
+.headerContainer {
+    width: 357px;
```
Comment (charliecog) [inline style.css:33]: again this is really specific, maybe a percentage is cleaner

File: style.css [hunk 9]
```diff
@@ -0,0 +1,171 @@
...
+}
+
+header h1 {
+    font-family: 'Marker Felt';
+    color: #000000;
+}
+
+#instructionsModalBtn {
+    height: 25px;
+    width: auto;
```
Comment (charliecog) [inline style.css]: potentially can remove

File: style.css [hunk 10]
```diff
@@ -0,0 +1,171 @@
...
+}
+section {
+    width: 100%;
```
Comment (charliecog) [inline style.css]: this is the default for sections already, could remove

File: style.css [hunk 11]
```diff
@@ -0,0 +1,171 @@
...
+.nextBtn button {
+    height: 40px;
+    width: 75px;
+    border-color: #B7B7B7;
+    border-radius: 20px;
+    margin-bottom: 15px;
+    background-color: #63666A;
+}
+
+/* The Modal (background) */
```
Comment (charliecog) [inline style.css]: remove all comments from the file

File: style.css [hunk 12]
```diff
@@ -0,0 +1,171 @@
...
+  .close {
+    color: #aaa;
+    float: right;
+    font-size: 28px;
+    font-weight: bold;
+  }
+  
+  .close:hover,
+  .close:focus {
+    color: black;
```
Comment (charliecog) [inline style.css]: use a hex code

**Review by charliecog — COMMENTED** [review]
General points:
- need to lock orientation to portrait as not displaying properly landscape on iphone se
- visually the next button is pretty hard to see (unless this is intentional for the time being)

File: index.html (hunk 1 above)
Comment (aidanmaskell) [reply index.html]: The lock button is 'disabled' as its default state, and then will be activated once the question has been answered

Comment (aidanmaskell) [reply index.html]: and so the button will be easier to see when 'active'

File: index.html (hunk 2 above)
Comment (aidanmaskell) [reply index.html]: comments removed

File: index.html (hunk 3 above)
Comment (aidanmaskell) [reply index.html]: naming changed to camelCase

File: style.css (hunk 5 above)
Comment (aidanmaskell) [reply style.css]: removed

File: style.css (hunk 6 above)
Comment (aidanmaskell) [reply style.css:20]: had a play around and the pixel width defined allows for no media screen features... would prefer to leave as it is, but we can discuss if you think its a really good idea :)

File: style.css (hunk 7 above)
Comment (aidanmaskell) [reply style.css]: removed

File: style.css (hunk 8 above)
Comment (aidanmaskell) [reply style.css:33]: see comment above relating to this

File: style.css (hunk 10 above)
Comment (aidanmaskell) [reply style.css]: removed

File: style.css (hunk 11 above)
Comment (aidanmaskell) [reply style.css]: removed

File: index.html [hunk 13]
```diff
@@ -0,0 +1,52 @@
...
+				<button id="answer1">Harry Potter and The Chamber of Secrets</button>
+				<button id="answer2">Shrek</button>
+				<button id="answer3">2 Fast 2 Furyush</button>
+			</div>
+			<div class="nextBtn">
+				<button id="next">Next</button>
+			</div>
+		</section>
+	</main>	
+	<p class="landscape">Landscape play is unsupported</p>
```
Comment (charliecog) [inline index.html]: remove this line

File: quiz.html (hunk 4 above)
Comment (charliecog) [reply quiz.html]: this is still here?

**Review by charliecog — COMMENTED** [review]
- landscape modal problems on iphone se

**Review by charliecog — APPROVED** [review]

**Pull request merged**